app/src/main/java/com/example/cleanarchitecturesolution/core/di/KoinInjector.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.cleanarchitecturesolution.core.di
+
+import org.koin.core.module.Module
+
+val koinInjector: List<Module> = listOf(
+    networkModule,
+    databaseModule,
+    appModule//is it right place?
```

Comment:
Tak, to dobre miejsce. `KoinInjector` ma nam agregować wszystkie moduły w naszej apce.